core/src/main/java/com/linecorp/armeria/common/ServerCacheControl.java

```diff
@@ -45,6 +45,7 @@ public final class ServerCacheControl extends CacheControl {
      * {@code "no-cache, no-store, must-revalidate"}.
```

Comment:
super minor nit, could also update the documentation

```suggestion
     * {@code "no-cache, no-store, must-revalidate, max-age=0"}.
```

Reply:
Should `max-age=0` be next to `no-cache` based on the building order?
```suggestion
     * {@code "no-cache, max-age=0, no-store, must-revalidate"}.
```

Reply:
Please ignore my review. I thought they are stringified with the building order.